===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(swTableColumn_add:col)) at 39 (line 87, column 10) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/swoole-src/src/memory/Table.c
+++ b/table1/benchmarks/swoole-src/src/memory/Table.c
@@ -86,6 +86,7 @@ int swTableColumn_add(swTable *table, char *name, int len, int type, int size)
     col->name = swString_dup(name, len);
     if (!col->name)
     {
+        free(col);
         return SW_ERR;
     }
     switch(type)
